quantum/quantum_keycodes.h

```diff
@@ -434,6 +434,18 @@ enum quantum_keycodes {
     // Shift, Enter
     KC_SFTENT,
 
+    // Left control, open paren
```

Comment:
You can move these above `KC_SFTENT` to be closer to the original keycodes. And there is probably no need to the extra blank lines since they're all part of the same feature.